src/set_theory/surreal.lean

```diff
@@ -253,9 +253,14 @@ theorem numeric_add : Π {x y : pgame} (ox : numeric x) (oy : numeric y), numeri
  end⟩
 using_well_founded { dec_tac := pgame_wf_tac }
 
--- TODO prove
--- theorem numeric_nat (n : ℕ) : numeric n := sorry
--- theorem numeric_omega : numeric omega := sorry
+/-- Pre-games defined by natural numbers are numeric. -/
+theorem numeric_nat : Π (n : ℕ), numeric n
+| 0 := numeric_zero
+| (n + 1) := numeric_add (numeric_nat n) numeric_one
+
+/-- The pre-game omega is numeric. -/
+theorem numeric_omega : numeric omega :=
+⟨by rintros ⟨ ⟩ ⟨ ⟩, λ i, by simp [numeric_nat i.down] , by rintros ⟨ ⟩⟩
```

Comment:
a style nit:
```suggestion
⟨by rintros ⟨⟩ ⟨⟩, λ i, by simp [numeric_nat i.down], by rintros ⟨⟩⟩
```
I'm curious - what is `simp` actually doing here? Scott's original proof was just `apply numeric_nat`. Does this work?
```suggestion
⟨by rintros ⟨⟩ ⟨⟩, λ i, numeric_nat i.down, by rintros ⟨⟩⟩
```

Reply:
This works! Using `simp` was just lazy on my part.
Thanks for cleaning up the code!